cmd/plasmacli/subcmd/keys/export.go

```diff
@@ -0,0 +1,55 @@
+package keys
+
+import (
+	"fmt"
+	"github.com/FourthState/plasma-mvp-sidechain/cmd/plasmacli/store"
+	"github.com/spf13/cobra"
+	"os"
+)
+
+// ExportCmd returns the keys export command
+func ExportCmd() *cobra.Command {
+	return exportcmd
+}
+
+var exportcmd = &cobra.Command{
+	Use:   "export <name> <location>",
+	Short: "Export a private key",
+	Long: `Exports a private key to a specified location (must be absolute path).
+Prints the address. 
+
+Usage:
+	plasmacli export <name> <location>
+
+The account is saved in encrypted format, you are prompted for a passphrase.
+You must remember this passphrase to unlock your account to export.
+`,
+	Args: cobra.MinimumNArgs(2),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		name := args[0]
+		location := args[1]
+
+		fd, err := os.OpenFile(location, os.O_RDWR|os.O_CREATE, 0755)
+		if err != nil {
+			return fmt.Errorf("error opening file: %s", err)
+		}
+		defer fd.Close()
+
+		accountjson, err := store.Export(name)
+
+		// Confirm write succeeds
+		bytestowrite := len(accountjson)
+		for bytestowrite > 0 {
```

Comment:
this seems unnecessary as per docs of `fd.Write` -> `Write writes len(b) bytes to the File. It returns the number of bytes written and an error, if any. Write returns a non-nil error when n != len(b). ` 

so if all of accountjson isn't written it errors, I think there are only two cases for this loop, it errors on first try or it succeeds and bytestowrite goes to 0 on entering second loop which is the equivalent of not having the loop and getting rid of the bytestowrite variable. 

I would replace lines 41-48 with something like:
```go
if numwritten, err := fd.Write(accountjson); err != nil {
	return fmt.Errorf("error writing to file: %s, bytes written: %s, total bytes to write: %s", err, numwritten, len(accountjson))
}
```

Reply:
Good point! The loop was a 162 pattern I got a little to familiar with lol